deployment_botpress/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "1"
+services:
+  botpress:
+    image: botpress/server
```

Comment:
I know we haven't done this in other place (but we should) but can you set the exact version (like we do with python packages) this was tested with. Else it will pull the latest build and once day weird things will happen and we won't know why.

Reply:
good idea, will add